napari/_qt/layer_controls/qt_labels_controls.py

```diff
@@ -248,6 +252,22 @@ def __init__(self, layer) -> None:
         self.renderComboBox = renderComboBox
         self.renderLabel = QLabel(trans._('rendering:'))
 
+        isoGradientComboBox = QComboBox(self)
+        grad_options = [i.value for i in IsoCategoricalGradientMode]
```

Comment:
Why this instead of QEnumComboBox from superqt?

Reply:
I did not know about superqt `QEnumComboBox`. I was matching code just above for the `rendering` combo box. I would rather not use two different classes here, but I'm happy to update both to use `QEnumComboBox` if that is preferred.